tests/safety/test_subaru.py

```diff
@@ -162,6 +163,30 @@ def _torque_cmd_msg(self, torque, steer_req=1):
     return self.packer.make_can_msg_panda("ES_LKAS", SUBARU_MAIN_BUS, values)
 
 
+class TestSubaruAngleSafetyBase(TestSubaruSafetyBase, common.AngleSteeringSafetyTest):
+  TX_MSGS = lkas_tx_msgs(SUBARU_MAIN_BUS, MSG_SUBARU_ES_LKAS_ANGLE)
+  RELAY_MALFUNCTION_ADDR = MSG_SUBARU_ES_LKAS_ANGLE
+  FWD_BLACKLISTED_ADDRS = fwd_blacklisted_addr(MSG_SUBARU_ES_LKAS_ANGLE)
+
+  FLAGS = Panda.FLAG_SUBARU_LKAS_ANGLE | Panda.FLAG_SUBARU_ES_STATUS
+
+  ANGLE_RATE_BP = [0, 5, 15]
+  ANGLE_RATE_UP = [5, 0.8, 0.15]
+  ANGLE_RATE_DOWN = [5, 3.5, 0.4]
+
+  def _angle_cmd_msg(self, angle, enabled=1):
+    values = {"LKAS_Output": angle, "LKAS_Request": enabled}
+    return self.packer.make_can_msg_panda("ES_LKAS_ANGLE", 0, values)
+
+  def _angle_meas_msg(self, angle):
+    values = {"Steering_Angle": angle}
+    return self.packer.make_can_msg_panda("Steering_Torque", 0, values)
+
+  def _pcm_status_msg(self, enable):
+    values = {"Cruise_Activated": enable}
+    return self.packer.make_can_msg_panda("ES_Status", self.ALT_CAM_BUS, values)
+
```

Comment:
So we've figured out that ES_Status->Cruise_Activated has none of the issues I *thought* I found?

Reply:
It doesn't appear to have the same issue:

https://github.com/commaai/panda/pull/1187#issuecomment-1641283653

Reply:
The EyeSight system must listen to at least a fault signal from the PCM, we should find that and log it all the time. Then log the fault bits from the camera only when not openpilotLongitudinalControl (as long as we can control with them set)

Reply:
when looking at gen2 CAN network diagram https://github.com/martinl/openpilot/wiki/CAN-Network-diagrams#2020my-outbacklegacy-can-diagram the message that contains the PCM fault signal (previously CruiseControl) should be sent by VDC CM on CAN1. I hope it helps narrowing it down